storage_service/locations/api/resources.py

```diff
@@ -475,6 +476,10 @@ def obj_create(self, bundle, **kwargs):
         if bundle.obj.package_type in (Package.AIP, Package.AIC, Package.DIP) and bundle.obj.current_location.purpose in (Location.AIP_STORAGE, Location.DIP_STORAGE):
             # Store AIP/AIC
             bundle.obj.store_aip(origin_location, origin_path)
+
+            # Asynchronously index AIP files
+            p = Process(target=bundle.obj.index_file_data_from_aip_mets)
+            p.start()
```

Comment:
Celery looks interesting: http://docs.celeryproject.org/en/latest/django/first-steps-with-django.html

It has Django integration, including the ability to store task results in the database using the Django ORM.